### PR TITLE
Added Tests for Bossy Calendar Controller

### PR DESCRIPTION
**Pull request by tuhintshukla**
This pull request includes a complete set of tests for bossy calendar controller.More tests to be added for bossy calendar directive.

### CODE REVIEW

**Comment by jasonmerino** [comment]
@tuhintshukla this branch needs master merged into it.

**Comment by tuhintshukla** [comment]
Sounds good will merge and submit again .

File: test/directives/bossy.calendar.spec.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    });
+
+    describe('Bossy Calender Directive',function(){
+
+        it('bossyCalendar should not be null',function(){
+
+            var dirConf={};
+            var x = compile('<bossy-calendar config="dirConf"></bossy-calendar>')(scope);
+            scope.$digest();
+            console.log(x);
```
Comment (jasonmerino) [inline test/directives/bossy.calendar.spec.js]: Let's remove `console.logs`.

File: test/directives/bossy.calendar.spec.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            expect(element).toNotBe(null);
+           // console.log(element.html());
```
Comment (jasonmerino) [inline test/directives/bossy.calendar.spec.js]: Let's remove commented out codes.

File: dist/js/bossy.all.js [hunk 3]
```diff
@@ -1397,6 +1399,7 @@ angular.module('bossy.tooltip', [])
                 // Disable browser's tooltip
                 element[0].title = '';
                 console.log(element);
```
Comment (jasonmerino) [inline dist/js/bossy.all.js]: Remove `console.log`.

**Comment by jasonmerino** [comment]
@tuhintshukla I made a couple of comments for code cleanup. Can you address those and then merge master into your branch again, then verify that `gulp jshint` returns no errors or warnings? Then we can merge!

**Comment by tuhintshukla** [comment]
Yes definitely.

Thanks,
Tuhin.

File: sites/sandbox/index.html [hunk 4]
```diff
@@ -47,6 +47,8 @@ <h3 class="panel-title">Option Settings</h3>
             </div>
         </div>
     </div>
+<<<<<<< HEAD
```
Comment (radfahrer) [inline sites/sandbox/index.html]: Looks like a bad merge here.

File: src/directives/temp.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```
Comment (radfahrer) [inline src/directives/temp.html]: we probably don't want to commit this file?

File: test/directives/bossy.data.spec.js [hunk 6]
```diff
@@ -1,29 +1,34 @@
-describe('bossyData',function(){
+xdescribe('bossyData',function(){
```
Comment (radfahrer) [inline test/directives/bossy.data.spec.js]: typo?

Comment (tuhintshukla) [inline test/directives/bossy.data.spec.js]: Not a typo, thats done to disable tests that are not valid.

and yes probably a bad merge. I will fix that and update the branch.

-Tuhin

File: test/directives/bossy.calendar.spec.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
+ddescribe('CalendarUnitTests for Directive', function() {
```
Comment (radfahrer) [inline test/directives/bossy.calendar.spec.js]: What's up with the ddescribe?

Comment (tuhintshukla) [inline test/directives/bossy.calendar.spec.js]: *ddescribe or iit* - run a single spec

*xdescribe or xit :- *disable running a particular spec

**Comment by tuhintshukla** [comment]
@radfahrer  This branch can be merged now.

**Comment by radfahrer** [comment]
@tuhintshukla is src/directives/temp.html needed?

**Comment by tuhintshukla** [comment]
No.

**Comment by radfahrer** [comment]
Looking good, @tuhintshukla, if you remove the temp.html file from this PR I'll merge it.

**Comment by tuhintshukla** [comment]
@radfahrer hopefully ,should be good now.  :)

**Pull request merged**